src/net.cpp

```diff
@@ -1099,9 +1099,9 @@ void V2Transport::ProcessReceivedMaybeV1Bytes() noexcept
     Assume(m_recv_state == RecvState::KEY_MAYBE_V1);
     // We still have to determine if this is a v1 or v2 connection. The bytes being received could
     // be the beginning of either a v1 packet (network magic + "version\x00"), or of a v2 public
```

Comment:
Now it reads `network magic + "version\x00" ... this 16-byte string`, but that's 12-byte string.

```suggestion
    // be the beginning of either a v1 packet (network magic + "version\x00\x00\x00\x00\x00"), or of a v2 public
```